dev_suites/dev_smart/smart_suite.rb

```diff
@@ -15,10 +17,121 @@ class SMARTSuite < Inferno::TestSuite
       request.query_parameters['state']
     end
 
-    group from: :smart_discovery
+    config options: {
+      redirect_uri: "#{Inferno::Application['inferno_host']}/custom/smart/redirect",
+      launch_uri: "#{Inferno::Application['inferno_host']}/custom/smart/launch"
+    }
 
-    group from: :smart_standalone_launch
+    group do
+      title 'Standalone Launch'
 
-    group from: :smart_ehr_launch
+      group from: :smart_discovery
+
+      group from: :smart_standalone_launch
+
+      group from: :smart_openid_connect,
+            config: {
+              inputs: {
+                id_token: { name: :standalone_id_token },
+                client_id: { name: :standalone_client_id },
+                requested_scopes: { name: :standalone_requested_scopes }
+              }
+            }
+
+      group from: :smart_token_refresh,
+            id: :smart_standalone_refresh_without_scopes,
+            title: 'SMART Token Refresh Without Scopes',
+            config: {
+              inputs: {
+                refresh_token: { name: :standalone_refresh_token },
+                client_id: { name: :standalone_client_id },
+                client_secret: { name: :standalone_client_secret },
+                received_scopes: { name: :standalone_received_scopes }
+              },
+              outputs: {
+                refresh_token: { name: :standalone_refresh_token },
+                received_scopes: { name: :standalone_received_scopes },
+                access_token: { name: :standalone_access_token },
+                token_retrieval_time: { name: :standalone_token_retrieval_time },
+                expires_in: { name: :standalone_expires_in }
+              }
+            }
+
+      group from: :smart_token_refresh,
+            id: :smart_standalone_refresh_with_scopes,
+            title: 'SMART Token Refresh With Scopes',
+            config: {
+              options: { include_scopes: true },
+              inputs: {
+                refresh_token: { name: :standalone_refresh_token },
+                client_id: { name: :standalone_client_id },
+                client_secret: { name: :standalone_client_secret },
+                received_scopes: { name: :standalone_received_scopes }
+              },
+              outputs: {
+                refresh_token: { name: :standalone_refresh_token },
+                received_scopes: { name: :standalone_received_scopes },
+                access_token: { name: :standalone_access_token },
+                token_retrieval_time: { name: :standalone_token_retrieval_time },
+                expires_in: { name: :standalone_expires_in }
+              }
+            }
+    end
+
+    group do
+      title 'EHR Launch'
```

Comment:
We can refine language here later, but to me this is a 'EHR Launch with OpenID Connect and Token Refresh End-to-End' test (Not just 'EHR Launch').  Which is a totally valid set of things to string together, but I wonder if we should present the individual tests as a group first, and separately have these types of integrated tests in a separate group?